src/components/Receive.vue

```diff
@@ -1,71 +1,52 @@
 <template>
   <div class="receive">
     <div>
-      <Card class="card" title="NEW PAYMENT REQUEST">
-        <div class="layout">
-          <div class="form">
-            <div class="row">
-              <div class="label">
-                <label class for>Label</label>
-              </div>
-              <Input v-model="label" />
-            </div>
-
-            <div class="submit">
-              <Button class="button" :onClick="generateAddress" type="primary"
-                >Generate address</Button
-              >
-            </div>
-          </div>
+      <div class="wrapper">
+        <div id="last-generated-address" class="address">
+          {{ addresses[addresses.length - 1].address }}
         </div>
-      </Card>
-
-      <Card class="card" title="GENERATED ADDRESSES">
-        <p class="address" v-for="address in addresses" :key="address.address">
-          {{ address.address }}
-        </p>
-      </Card>
-    </div>
-    <div class="text">
-      <p class="title">About payment requests</p>
-      <p class="paragraph">
-        Every time you generate a receiving address in Sheikah, you can label and store it as a
-        payment request. This allows you to better trace every payment you receive.
-      </p>
-      <p class="paragraph">
-        You can generate as many payment requests at once. However, it is advised that you only
-        generate as you need them, instead of creating a lot of them preemptively.
-      </p>
-
-      <p class="paragraph">
-        All this data is completely private and it is only stored in your device in a encrypted
-        database that only you can read.
-      </p>
+        <div :class="[copied ? 'copied' : 'copy']" @click="copyAddress()">
+          {{ text }}
+        </div>
+      </div>
     </div>
   </div>
 </template>
 
 <script>
-import Card from './card/Card'
-import Button from './Button'
-import Input from './Input'
-
 export default {
   name: 'receive',
   beforeCreate() {
     this.$store.dispatch('getAddresses')
   },
-  components: {
-    Card,
-    Button,
-    Input,
-  },
   data() {
     return {
       label: '',
+      copied: false,
+      text: 'Copy',
     }
   },
   methods: {
+    copyAddress() {
```

Comment:
IMHO, the copy logic could be moved to a copy function in utils and be called from here. 

```suggestion
    copyAddress() {
      const idToCopy = 'last-generated-address'
      this.copied = copy(idToCopy)
    }
```